komodo/symlink/suggester/configuration.py

```diff
@@ -49,6 +49,11 @@ def update(self, release, mode):
                 else:
                     self.links[link] = repr(release)
         elif mode == "stable":
+            # Add/update deprecated link iff stable exists and has changed
+            if self.links.get(link) != release.month_alias():
+                dlink = link.replace("stable", "deprecated")
+                self.links[dlink] = self.links[link]
+
             self.links[release.month_alias()] = repr(release)
```

Comment:
Well, this question doesn't belong to your PR, but why it's called `month_alias` when it returns essential `Y.M-PYVER` ?

Reply:
Because `month_alias` is the alias to the actual release and represents the month. @jondequinor can probably answer better.

Reply:
Yeah, this is correct. No release should be without the python version postfix.